CHANGELOG.md

```diff
@@ -3,52 +3,99 @@
 `relude` uses [PVP Versioning][1].
 The changelog is available [on GitHub][2].
 
-## Unreleased: 0.6.0.0
+## 0.6.0.0 — Oct 30, 2019
 
-* [#190](https://github.com/kowainik/relude/pull/190):
-  Return Failure early in `<*` and `*>` too.
-* [#170](https://github.com/kowainik/relude/issues/170):
-  Implement `Elem` type family!
-* [#203](https://github.com/kowainik/relude/issues/203):
-  Implement `guarded` takes a predicate and a value and returns an Alternative.
-* [#201](https://github.com/kowainik/relude/issues/201):
-  Implement `!!?` as a safe equivalent of `!!` that returns a `Maybe`!
-* [#192](https://github.com/kowainik/relude/issues/192):
-  Reexport `foldMap'` from `Data.Foldable`.
-* [#187](https://github.com/kowainik/relude/issues/187):
-  Remove `tasty` and `tasty-hedgehog` dependencies and their redundant imports.
-* [#195](https://github.com/kowainik/relude/pull/195):
-  Implement `foldMap1` for `NonEmpty` in terms of `foldr`.
-* [#194](https://github.com/kowainik/relude/pull/194):
-  Use `mapToFst` instead of `zip` to improve list fusion in `inverseMap`.
-* [#193](https://github.com/kowainik/relude/pull/193):
-  Implement `andM`, `orM`, `allM`, and `anyM` in terms of `&&^` and `||^`.
-* [#191](https://github.com/kowainik/relude/pull/191):
-  Implement `asumMap` and `foldMapA` by coercing `foldMap`.
-  BREAKING CHANGE: Reorder type parameters to `asumMap`.
-* Use `$>` instead of `*>` and `pure` where possible.
-* [#167](https://github.com/kowainik/relude/issues/167):
-  Rename functions `prec`/`prev`, `dupe`/`dup`.
-* [#155](https://github.com/kowainik/relude/issues/155):
-  Implement `Relude.Extra.Foldable` module.
-* Re-export `GHC.Float.atan2`.
-* [#172](https://github.com/kowainik/relude/issues/172):
-  Add `Monoid` and `Semigroup` instances for `Validation` type
-* [#177](https://github.com/kowainik/relude/issues/177):
-  Improve usage of performance pragmas.
-* [#178](https://github.com/kowainik/relude/issues/178):
-  Made `die` be polymorphic in its return type.
+* [#171](https://github.com/kowainik/relude/issues/171):
+  Add custom type errors to various functions and instances.
```

Comment:
Do you think is worth mentioning in which places?

Reply:
Good idea! It's the most important issue in this release IMHO. Should I specify every place or just describe shortly like _for functions that similar with Prelude but have different behavior_?

Reply:
hmm, I would prefer to specify exact places, it could be just enumeration, I think. What would you prefer?

Reply:
@vrom911 It actually looks pretty good :+1:  I've enumerated all places and I also grouped them so the list doesn't become too long. Check it out :slightly_smiling_face: